.github/workflows/php-test.yml

```diff
@@ -44,6 +44,8 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: npm
+      - name: Composer update
+        run: composer update
```

Comment:
Why is this addition needed?

Reply:
Thanks, @felixarntz for the review and question.

Without `composer update` it gives the same error [shown here](https://github.com/WordPress/performance/actions/runs/3653782252/jobs/6173598812) as it did not pull updated composer packages.

similar issue replicated on local systems:
1. git clone git@github.com:WordPress/performance.git
2. git checkout fix/unit-tests-failure
3. npm run test-php

If we run `composer update` before step 3, then unit testing works fine.

Reply:
Thanks @mukeshpanchal27! I still gave this another try, I was thinking maybe it's due to some cache of Composer dependencies. I'm still a bit confused why it is necessary, but you're right that it seems needed for the tests to pass. Let's leave it as is for now 👍